lite/kernels/npu/graph_compute.cc

```diff
@@ -30,10 +30,14 @@ void GraphCompute::PrepareForRun() {
   auto& ctx = this->ctx_->template As<NPUContext>();
   auto& param = this->Param<param_t>();
 
-  exec_ = ctx.client(param.model_name);
-  CHECK(exec_);
+  CHECK(param.weight);
+  CHECK(lite::npu::LoadModel(*param.weight, &model_client_, &model_name_));
+  // TODO(hong19860320): find an good way to free the model data.
+  param.weight->Resize({1});
+  param.weight->mutable_data(TargetType::kARM, 1);
```

Comment:
what is the 1 stand for here

Reply:
I will change it to 'param.weight->mutable_data<int8_t>(TargetType::kARM)'